File: server/src/main/java/dev/slimevr/config/VMCConfig.java
package dev.slimevr.config;


public class VMCConfig extends OSCConfig {

	// Anchor the tracking at the hip?
	boolean anchorHip = true;
	// Address of the VRM to be used
	String VRMAddress = "";

	public VMCConfig() {
	}

	public boolean getAnchorHip() {
		return anchorHip;
	}

	public void setAnchorHip(boolean anchorHip) {
		this.anchorHip = anchorHip;
	}

	public String getVRMAddress() {
		return VRMAddress;
	}

	public void setVRMAddress(String VRMAddress) {
		this.VRMAddress = VRMAddress;
	}
}

File: server/src/main/java/dev/slimevr/config/VMCConfig.kt
package dev.slimevr.config

import kotlinx.serialization.Serializable

@Serializable
class VMCConfig : OSCConfig() {

	// Anchor the tracking at the hip?
	var anchorHip = true

	// JSON part of the VRM to be used
	var vrmJson: String? = null
}
